src/Widgets/LogInButtonWidget/LogInButtonWidgetEditingConfig.ts

```diff
@@ -1,9 +1,9 @@
 import { provideEditingConfig } from 'scrivito'
-import { LoginButtonWidget } from './LoginButtonWidgetClass'
+import { LogInButtonWidget } from './LogInButtonWidgetClass'
 import Thumbnail from './thumbnail.svg'
 
-provideEditingConfig(LoginButtonWidget, {
-  title: 'LoginButton',
+provideEditingConfig(LogInButtonWidget, {
+  title: 'Log In Button',
```

Comment:
* [ ] Actually it is a "login button" a button for the login (noun). If we want to stick with the verb "to log in" it would be a "log-in button" but that's not commonly used.

Reply:
See https://github.com/Scrivito/scrivito-portal-app/pull/404